### DISCUSSION TITLE
[PLUGIN] Mark font red  of the position being browsed in the Table of contents.

### DISCUSSION THREAD

**Discussion by SiriusXT** [TriliumNext/Trilium#3707] [Show and tell]
Mark font red  of the position being browsed in the Table of contents.
https://github.com/SiriusXT/trilium-show-position-in-toc

![preview](https://user-images.githubusercontent.com/37627919/225192622-59496680-3a77-4849-b55a-57bc6b308483.png)

**Comment by Nriver** [comment]
The toc looks more great now! Thanks for your creation.
